grails-app/controllers/com/recomdata/transmart/data/association/GeneprintController.groovy

```diff
@@ -26,6 +26,9 @@ class GeneprintController {
 
     AnalysisConstraints analysisConstraints
 
+    double mrnaThreshold;
```

Comment:
Although a controller class could be instantiated per http request it would be safer to assume that it's not and prefer to use local variables.